packages/components/src/components/Snackbar/Snackbar.tsx

```diff
@@ -11,16 +11,16 @@ import {
   useWindowDimensions,
 } from 'react-native'
 import { ToastProps } from 'react-native-toast-notifications/lib/typescript/toast'
+import { ToastType } from 'react-native-toast-notifications'
 import { useTranslation } from 'react-i18next'
 import React, { FC } from 'react'
-import Toast, { ToastType } from 'react-native-toast-notifications'
-import ToastContainer from 'react-native-toast-notifications'
 
 import { Icon, IconProps } from '../Icon/Icon'
 import { Spacer } from '../Spacer/Spacer'
 import { useTheme } from '../../utils'
 
-// let snackbarOffset: number = 10
+// TODO: Replace with global setting
+export const DEFAULT_OFFSET: number = 50
```

Comment:
Potentially should add `SNACKBAR_` or maybe even `COMPONENTS_SNACKBAR_` or something since if we're defining it flat I think it could also conflict any global variables defined by an app AKA would be safest to be named obnoxiously specifically to prevent any conflict.

Reply:
Also, leaving the offset as 50 was just random(ly approximately above the bottom bar in Storybook) since I knew another ticket would follow. I think the default offset should probably be 0 or something more thoughtful. Maybe slightly more than 0 because I think with the keyboard open it then is right against the top of it? Perhaps a Jessica call there, but maybe ensuring a minimum 4 or 8 pixels off the keyboard would be good.

Not sure if we should inherently factor in OS. The offsets in flagship look to be: 
- iOS
   - 25
   - 94 w/ Nav
- Android
   - 0
   - 66 w/ Nav

Not sure if we should have any iOS/Android differences though and leave it to the consumer to manipulate as appropriate from default.

Potentially relevant note: in flagship they are using SafeAreaView which is an iOS thing, but I did not include it because it seemed like the library was handling well enough to keep it on screen and it seemed superfluous. That behavior may have altered what flagship is doing with the numbers above.

Reply:
Getting around to watching the videos, maybe I am wrong about the `SafeAreaView` part above. I see your iOS device has the bottom bar thing that mine does not (my test device having a physical home button and yours having the bar since it doesn't?). 

Maybe we should leave the default for this ticket and split out another ticket to tackle ensuring we're handling device specific offsets well?